MAUI/NavigationDrawer/Side-Pane-Content.md

```diff
@@ -0,0 +1,363 @@
+---
+layout: post
+title: Setting Sliding Panel Content in .NET MAUI NavigationDrawer | Syncfusion
+description: Learn here all about Setting Sliding Panel Content support in Syncfusion .NET MAUI Navigation Drawer (SfNavigationDrawer) control and more.
+platform: maui
+control: NavigationDrawer
+documentation: ug
+---
+
+
+# Setting Sliding Panel Content in .NET MAUI Navigation Drawer
+
+The drawer pane content is viewable only if the drawer is in the open state. Its content can be set as
```

Comment:
The drawer pane content is only viewable when the drawer is in the open state. Its content can be set as

Reply:
Changed